policy/common/methods.go

```diff
@@ -23,9 +23,11 @@ import (
 )
 
 type Methods struct {
-	Comments        []string `yaml:"comments,omitempty"`
-	CommentPatterns []Regexp `yaml:"comment_patterns,omitempty"`
-	GithubReview    bool     `yaml:"github_review,omitempty"`
+	Comments                    []string `yaml:"comments,omitempty"`
+	CommentPatterns             []Regexp `yaml:"comment_patterns,omitempty"`
+	GithubReview                bool     `yaml:"github_review,omitempty"`
+	GithubReviewComments        []string `yaml:"github_review_comments,omitempty"`
```

Comment:
Let's remove this option and only support `GitHubReviewCommentPatterns`.

If I were to implement Policy Bot again (or make a v2 configuration format), I think I'd only support `CommentPatterns`, since it's a superset of the plain `Comments` field. While we haven't formally deprecated `Comments`, I consider it deprecated.

Reply:
makes sense, fine by me.